stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/BackdropComponent.kt

```diff
@@ -57,7 +58,7 @@ fun Backdrop(
                 settingsUiState.deliverToLabel()?.asString(),
                 themeColor,
                 launchBottomSheet = {
-                    if (manageStockViewModel.canReview()) {
+                    if (manageStockViewModel.dataEntryUiState.value.hasUnsavedData) {
```

Comment:
you can use `dataEntryUiState.hasUnsavedData` property

Reply:
yes, indeed